### DISCUSSION TITLE
Inter Panel Interaction

### DISCUSSION THREAD

**Discussion by anujms1999** [grafana/grafana#38384] [Dashboards, Panels & Visualizations]
Currently there is no way we can update a variable on an event caused in say some x panel and use the updated variable in say y panel. 
My suggestion would be to include such a feature that inter panel communication is allowed.
This is an important feature because while monitoring there are cases where we want tables of those displayed using single stat. Though variable serves the function but instead of that if someone goes by the path of showing all variables and then wanting to get a table on a click in the same dashboard is impossible.

**Comment by lstroud** [comment]
+1

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
